launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue

```diff
@@ -202,6 +212,46 @@ const getKeyState = computed(() => {
   }
 });
 
+const getKeyHanlingTime = computed(() => {
```

Comment:
Not quite sure if this is a typo? :thinking: 

```suggestion
const getKeyHandlingTime = computed(() => {
```

Reply:
nice catch!